flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -1029,6 +1029,8 @@ private TableResultInternal executeInternal(
                         defaultJobName,
                         jobStatusHookList);
         try {
+            ClassLoader userClassLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
please could you change the variable name to be something like originalContextClassLoader

Reply:
Changed the variable name to `contextClassLoader` .